### ISSUE TITLE
Line up Issue with Added Languages

### ISSUE THREAD

**Issue by Dirty-Butter** [cubecart/v6#44]
Checkbox, Edit, Trashcan, Download on right column does not line up with default UK language.

**Comment by bhsmither** [comment]
In the admin template settings.language.php, line 15-17, make the edit on line 17:
```
	<h3>{$LANG.translate.title_languages}</h3>
	<div>
	<div class="lang_item">
```
Then, in layout.css (anywhere that makes sense), add:
```
.lang_item {
    margin-top: 5px;
}
```
Personally, I would like to have a zebra-table.

**Comment by Dirty-Butter** [comment]
Not working - suspect nested div issue.

**Comment by bhsmither** [comment]
I have noticed that my browser needs to be forced to reload page resources. For Firefox, that would be holding down the shift key when clicking the Reload Current Page icon. Maybe your browser hasn't fetched a fresh copy of the CSS file.

I would also say to disable the cache mechanism while playing around with the skins, but I can't find the setting.

Or maybe I misinterpreted what you initially described. I was working on a malformed listing similar to issue #26.
